worker/src/scripts/upsertDefaultModelPrices.ts

```diff
@@ -0,0 +1,187 @@
+        ) {
+          logger.debug(
+            `Default model ${defaultModelPrice.model_name} (${defaultModelPrice.id}) already up to date. Skipping.`
+          );
+          continue;
+        }
+
+        // Upsert model and prices in a transaction
+        promises.push(
+          prisma
+            .$transaction(async (tx) => {
+              await tx.model.upsert({
+                where: {
+                  projectId: null,
+                  id: defaultModelPrice.id,
+                },
+                update: {
+                  modelName: defaultModelPrice.model_name,
+                  matchPattern: defaultModelPrice.match_pattern,
+                  updatedAt: defaultModelPrice.updated_at,
+                  tokenizerConfig:
+                    defaultModelPrice.tokenizer_config ?? undefined,
+                  tokenizerId: defaultModelPrice.tokenizer_id,
+                },
+                create: {
+                  projectId: null,
+                  id: defaultModelPrice.id,
+                  modelName: defaultModelPrice.model_name,
+                  matchPattern: defaultModelPrice.match_pattern,
+                  tokenizerConfig:
+                    defaultModelPrice.tokenizer_config ?? undefined,
+                  tokenizerId: defaultModelPrice.tokenizer_id,
+                  createdAt: defaultModelPrice.created_at,
+                  updatedAt: defaultModelPrice.updated_at,
+                },
+              });
+
+              for (const [itemName, price] of Object.entries(
+                defaultModelPrice.prices
+              )) {
+                await tx.price.upsert({
+                  where: {
+                    modelId_itemName: {
+                      modelId: defaultModelPrice.id,
+                      itemName,
+                    },
+                  },
+                  update: {
+                    price,
+                    updatedAt: defaultModelPrice.updated_at,
+                  },
+                  create: {
+                    modelId: defaultModelPrice.id,
+                    itemName,
+                    price,
+                    createdAt: defaultModelPrice.created_at,
+                    updatedAt: defaultModelPrice.updated_at,
+                  },
+                });
+              }
+
+              logger.info(
+                `Upserted default model ${defaultModelPrice.model_name} (${defaultModelPrice.id})`
+              );
+            })
+            .catch((error) => {
+              logger.error(
+                `Error upserting default model ${defaultModelPrice.model_name} (${defaultModelPrice.id}): ${error.message}`,
+                {
+                  error,
+                }
+              );
+            })
+        );
+      }
+
+      await Promise.all(promises);
+      logger.debug(`Completed batch ${i + 1} of ${numBatches}.`);
+    }
+
+    logger.debug("Finished upserting default model prices.");
```

Comment:
Can you make this an info logger and add latency here how much time this function took?
Also, please do the same in the catch block below.

Reply:
Done!